pkg/agent/openflow/pipeline.go

```diff
@@ -3011,15 +3007,15 @@ func (f *featureService) gatewaySNATFlows() []binding.Flow {
 			pktDstRegMarks = append(pktDstRegMarks, ToUplinkRegMark)
 		}
 		for _, pktDstRegMark := range pktDstRegMarks {
-			// This generates the flow to match the first packet of NodePort / LoadBalancer connection initiated through the
-			// Antrea gateway and externalTrafficPolicy of the Service is Cluster, and the selected Endpoint is on a remote
-			// Node, then ConnSNATCTMark will be loaded in DNAT CT zone, indicating that SNAT is required for the connection.
+			// This generates the flow to match the first packet of Service which is accessible externally connection
+			// initiated through the Antrea gateway, and the selected Endpoint is on a remote Node, then ConnSNATCTMark
+			// will be loaded in DNAT CT zone, indicating that SNAT is required for the connection.
```

Comment:
The comment's readability is not good. "This generates the flow to match the first packets of externally-originated connections towards external addresses of the Service..."

Reply:
Done.